src/Network/Receive.pm

```diff
@@ -4068,11 +4068,14 @@ sub npc_store_info {
 		my $ID = unpack("v1", substr($msg, $i + 9, 2));
 
 		my $store = $storeList[$storeList] = {};
-		# TODO: use itemName() or itemNameSimple()?
-		my $display = ($items_lut{$ID} ne "")
-			? $items_lut{$ID}
-			: T("Unknown ").$ID;
-		$store->{name} = $display;
+		
+		my $name = itemNameSimple($ID);
```

Comment:
Because itemName requires an Actor::Item as parameter and npc item lists do not create that, it could be used if npcs properly used inventoryList and Actor::Item